enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -46,13 +47,15 @@ type InteractiveQuestionProviderProps = PropsWithChildren<
 
 export const InteractiveQuestionProvider = ({
   cardId,
-  options,
+  options = {},
```

Comment:
This will create new object on every render which may affect memoization down the line. 

```suggestion
const DEFAULT_OPTIONS: QueryParams = {};

export const InteractiveQuestionProvider = ({
  cardId,
  options = DEFAULT_OPTIONS,
```

Reply:
i love react